### PR TITLE
Database preparation docs improvements

### PR DESCRIPTION
**Pull request by jfy133**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/funcscan/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/funcscan _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by jfy133** [comment]
@nf-core-bot fix linting

File: docs/usage.md [hunk 1]
```diff
@@ -360,15 +431,22 @@ The same applies for the antiSMASH installation directory, which is also a requi
 
 To supply the database directories to the pipeline:
 
-1. Install antiSMASH from [bioconda](https://bioconda.github.io/recipes/antismash-lite/README.html)
-2. Run `download-antismash-databases`
+1. Install antiSMASH from [bioconda](https://bioconda.github.io/recipes/antismash-lite/README.html) (To ensure database compatibility, please use the same version as is used in your nf-core/funcscan release - check version in file `<pipeline_installation>/<path>/funcscan/modules/nf-core/amrfinderplus/run/environment.yml`)
```
Comment (jasmezz) [inline docs/usage.md:434]:
```suggestion
1. Install antiSMASH from [bioconda](https://bioconda.github.io/recipes/antismash-lite/README.html). To ensure database compatibility, please use the same version as is used in your nf-core/funcscan release (check version in file `<pipeline_installation>/<path>/funcscan/modules/nf-core/antismash/antismashlite/environment.yml`).
```

File: docs/usage.md [hunk 2]
```diff
@@ -360,15 +431,22 @@ The same applies for the antiSMASH installation directory, which is also a requi
...
+
+```bash
+conda create -n antismash-lite -c bioconda antismash-lite
+conda activate antismash-lite
+```
+
+2. Run the following command `download-antismash-databases`. Use `--database-dir` to specify a new location.
```
Comment (jasmezz) [inline docs/usage.md:441]:
```suggestion
2. Run the command `download-antismash-databases`. Use `--database-dir` to specify a new location.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -61,6 +61,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#406](https://github.com/nf-core/funcscan/pull/406) Fixed prediction tools not being executed if annotation workflow skipped. (by @jasmezz)
 - [#407](https://github.com/nf-core/funcscan/pull/407) Fixed comBGC bug when parsing multiple antiSMASH files. (by @jasmezz)
 - [#409](https://github.com/nf-core/funcscan/pull/409) Fixed argNorm overwriting its output for DeepARG. (by @jasmezz, @jfy133)
+- [#412](https://github.com/nf-core/funcscan/pull/412) Improve all pre-run database download documentation (by @jfy133)
```
Comment (jasmezz) [inline CHANGELOG.md:64]:
```suggestion
- [#412](https://github.com/nf-core/funcscan/pull/412) Improve all pre-run database download documentation. (by @jfy133)
```

**Review by jasmezz — APPROVED** [review]

**Pull request merged**